docs/sources/auth/planning/index.md

```diff
@@ -0,0 +1,104 @@
+
+Teams is a perfect solution to work with a small subset of users. Teams can share resources among other teams.
+
+### Users in Grafana Organizations
+
+[Grafana Organizations] was born as a concept to isolate users from dashboards and datasources by having multiple organizations under a single instance of Grafana. This means that users under different organizations won't share any resources such as dashboards, folders, and datasources.
+
+Grafana Organizations provide a mesure of isolation within Grafana by default. The intention is to present different user experiences, which give the sense of different instances of Grafana within a single instance. However, **we recommend aginast Grafana Organizations** because they lack the scalability of [Folders].
+
+### Choosing between teams and organizations
+
+Grafana Teams and Grafana Organizations serve similar purposes in the Grafana platform. Both are designed to help manage and control access to resources, Teams provide a more focused approach for smaller groups or projects. Teams allow for easy collaboration and shared ownership of resources among members, making it simple to manage access and control on who has access to said resources.
+
+Organizations, on the other hand, provide a higher level of management for multiple teams and resources, and are designed for larger enterprises or organizations with a complex setup or multiple and different business models.
+
+That being said, Teams provide many of the same benefits as Organizations, and can be seen as a more streamlined and simplified approach to managing resources and access. As such, it is possible that in the future, Teams may eventually replace Organizations as the primary way to manage resources in Grafana.
+
+## Do I have external systems?
+
+Consider the need for [M2M] communications. If there's a system that needs to interact with Grafana, ensure that it has proper access for it.
+
+Here are some example scenarios:
+
+**IoT devices monitoring**: Sensors and actuators might want to ingest information to Grafana in an automatic way. Consider the security implications of a shared security access for all the devices or an individual access to each of them.
+
+**Network monitoring**: Having a distributed systems architecture performance reported back to Grafana might provide insight about bottlenecks and even trigger alerts in the need of a timely intervention.
+
+These are just a few examples of how Grafana can be used in M2M scenarios. The platform is highly flexible and can be used in a variety of other M2M applications, making it a powerful tool for organizations looking to gain insights into their systems and devices.
```

Comment:
@linoman , I would add one more scenario. Users like things that come in 3s. Also, do we have any blog posts that we can link the reader to for each of these scenarios? They might want to read up on a use case.

Reply:
Changed.